projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -66,6 +66,32 @@ function get_checklist_definitions() {
 	);
 }
 
+/**
+ * Determines whether or not design selected task is enabled
+ *
+ * @return boolean True if design selected task is enabled
+ */
+function can_update_design_selected_task() {
+	$site_intent = get_option( 'site_intent' );
+	return $site_intent === 'free' || $site_intent === 'build' || $site_intent === 'write';
+}
```

Comment:
This one looks like a bad merge, since isn't related with the PR, right?

Reply:
Whoops, must have forgot to rebase after switching the base branch. Fixed that! :+1: